tests/datacatalog_tag_manager/tag_datasource_processor_test.py

```diff
@@ -120,7 +149,7 @@ def test_create_tags_from_csv_column_metadata_should_succeed(self, mock_read_csv
         self.assertFalse('string_field' in created_tag_2.fields)
 
     def test_create_tags_from_csv_permission_denied_lookup_entry_should_skip_resource(
-        self, mock_read_csv):  # noqa
+            self, mock_read_csv):  # noqa
```

Comment:
Was this change performed by any tool? The current version is passing in the CI pipeline, so I didn't understand why we need to do this...

Reply:
I ran yapf, and it was formatted by it... weird, reverted it